portfolio/src/main/webapp/script.js

```diff
@@ -25,4 +25,13 @@ function addRandomGreeting() {
   // Add it to the page.
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
+  
 }
+async function showServerTime() {
+    const responseFromServer = await fetch('/hello');
+    const textFromResponse = await responseFromServer.text();
+  
+    const dateContainer = document.getElementById('date-container');
+    dateContainer.innerText = textFromResponse;
+  }
```

Comment:
[4.2](https://google.github.io/styleguide/jsguide.html#formatting-block-indentation): Each time a new block or block-like construct is opened, the indent increases by two spaces.

Once you've set the editor's default spacing by setting **Editor: Tab Size** to 2 via **File** > **Preferences** > **Open Settings (UI)**. And then, to format a file:
1. Open the Command Palette using `Ctrl`/`Cmd` + `Shift` + `P`.
1. Find and invoke **Format Document**.